jmix-flowui/flowui/src/main/java/io/jmix/flowui/data/aggregation/impl/BasicAggregation.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.data.aggregation.impl;
+
+import io.jmix.flowui.component.AggregationInfo;
+import io.jmix.flowui.data.aggregation.Aggregation;
+import org.springframework.lang.Nullable;
+
+import java.util.Collection;
+import java.util.EnumSet;
+
+public class BasicAggregation<T> implements Aggregation<T> {
```

Comment:
What a weird word `basic`. Rename all `BasicXxxAggregation` to `AbstractXxxAggregation`. 
Create `CountAggregation extends AbsAgr` for the corresponding types.